src/ape/_cli.py

```diff
@@ -20,14 +20,15 @@
 
 
 def display_config(ctx, param, value):
+
```

Comment:
This is really nitty but I don't like the newline separating the function signature and the first line of the function